third_party/2and3/redis/client.pyi

```diff
@@ -261,19 +261,19 @@ class Redis(object):
     def pfadd(self, name, *values): ...
     def pfcount(self, name): ...
     def pfmerge(self, dest, *sources): ...
-    def hdel(self, name, *keys): ...
-    def hexists(self, name, key): ...
-    def hget(self, name, key): ...
-    def hgetall(self, name): ...
-    def hincrby(self, name, key, amount=...): ...
-    def hincrbyfloat(self, name, key, amount=...): ...
-    def hkeys(self, name): ...
-    def hlen(self, name): ...
-    def hset(self, name, key, value): ...
-    def hsetnx(self, name, key, value): ...
-    def hmset(self, name, mapping): ...
-    def hmget(self, name, keys, *args): ...
-    def hvals(self, name): ...
+    def hdel(self, name: _Key, *keys: Iterable[_Key]) -> int: ...
+    def hexists(self, name: _Key, key: _Key) -> bool: ...
+    def hget(self, name: _Key, key: _Key) -> Optional[bytes]: ...
+    def hgetall(self, name: _Key) -> Dict[bytes, bytes]: ...
+    def hincrby(self, name: _Key, key: _Key, amount: int = ...) -> int: ...
+    def hincrbyfloat(self, name: _Key, key: _Key, amount: float = ...) -> float: ...
+    def hkeys(self, name: _Key) -> List[bytes]: ...
+    def hlen(self, name: _Key) -> int: ...
+    def hset(self, name: _Key, key: _Key, value: _Str) -> int: ...
+    def hsetnx(self, name: _Key, key: _Key, value: _Str) -> int: ...
+    def hmset(self, name: _Key, mapping: Mapping[_Str, _Str]) -> bool: ...
+    def hmget(self, name: _Key, keys: Iterable[_Key], *args: Iterable[_Key]) -> List[Optional[bytes]]: ...
```

Comment:
If I read the code correctly (especially the `list_or_args()` method), `keys` can either be `str` or `bytes` or an iterable. I think this is correct:

```suggestion
    def hmget(self, name: _Key, keys: Union[_Key, Iterable[_Key]], *args: _Key) -> List[Optional[bytes]]: ...
```